src/markdown_generators.js

```diff
@@ -161,24 +181,11 @@ const generateMarkdownRecursive = (page, courseUidsLookup, courseData) => {
   }
 }
 
-const generateCourseInfo = courseData => ({
```

Comment:
I removed this because it looked like it was unused, but I'm happy to add back if it's needed

Reply:
You are correct, and I think there is other unused code that can be cleaned up as well